specs/wp-plan-purchase-spec.js

```diff
@@ -68,7 +68,7 @@ describe( `[${ host }] Plans: (${ screenSize }) @parallel @jetpack`, function()
 		}
 	} );
 
-	describe( 'Viewing a specific plan with coupon:', function() {
+	describe.only( 'Viewing a specific plan with coupon:', function() {
```

Comment:
Was this `.only` intentional for testing purposes? Just in case I'll leave a comment so we don't forget it :)

Reply:
🤦‍♂️ yeah that was my testing purposes, pushed a commit removing it.